src/wiseagents/wise_agent_impl.py

```diff
@@ -321,6 +321,69 @@ def _get_retrieval_query(self) -> str:
         """
 
 
+class SequentialCoordinatorWiseAgent(WiseAgent):
+    """
+    This agent will coordinate the execution of a sequence of agents.
+    Use Stomp protocol.
+    """
+    yaml_tag = u'!wiseagents.SequentialCoordinatorWiseAgent'
+
+    def __init__(self, name: str, description: str, transport: WiseAgentTransport, agents: List[str]):
+        self._name = name
+        self._agents = agents
```

Comment:
I'm going to merge this PR as is, but what about to add this index as part of the context and take it from there? This way, potentially, also other agents or external actors could add steps. For example, imagine an agent that based on an LLM answer decide another call to RAG is needed to get the final answer. It could add another step and the coordinator will take care of invoke the step from the modified sequence. Does it make sense? Maybe we can add a subtask to the related issue or adding a new one as improvement